.eleventy.js

```diff
@@ -11,7 +17,32 @@ module.exports = (config) => {
   })
 
   config.addPlugin(syntaxHighlight)
-  config.addPlugin(lazyImagesPlugin)
+
+  config.addShortcode("image", async function(src, alt) {
+    if(alt === undefined) {
+      throw new Error(`Missing \`alt\` on image from: ${src}`);
+    }
+
+    let outputFormat = extension(src) || "jpeg"
+
+    let stats = await Image(src, {
```

Comment:
I generally prefer using `const` instead of `let` if re-assignment is not needed.

Reply:
Agreed. I took this from the documentation and did not bother changing the variable declaration style.

Reply:
I amended the relevant commit.